bigchaindb/upsert_validator/validator_election.py

```diff
@@ -114,7 +117,7 @@ def validate(self, bigchain, current_transactions=[]):
         if not self.is_same_topology(current_validators, self.outputs):
             raise UnequalValidatorSet('Validator set much be exactly same to the outputs of election')
 
-        return True
+        return self
```

Comment:
Why did you change it?

Do not forget to update the docstring as well or perhaps remove that part of the docstring to avoid the outdated documentation.